mod/settings.php

```diff
@@ -1277,7 +1277,7 @@ function settings_content(App $a) {
 
 		'$h_pass' 	=> t('Password Settings'),
 		'$password1'=> array('password', t('New Password:'), '', ''),
-		'$password2'=> array('confirm', t('Confirm:'), '', t('Leave password fields blank unless changing')),
```

Comment:
>  I wonder was in the past the changing password section linked to any of the other sub-sections in these settings

In the past, before the sections were collapsed, everything was shown at the same time. Maybe now with the collapsed sections the hint is redundant for experienced internet users. But I'm for keeping it in.